requirements.txt

```diff
@@ -1,3 +1,4 @@
 altair
 portpicker
 tornado<6.0
+jupyter-server-proxy; python_version >= '3.5'
```

Comment:
I'm concerned about unconditionally adding this as a dependency, as it is a pretty heavy one (it depends on essentially the entire jupyter ecosystem). Can we remove this from requirements and perhaps just document that it is required if using the proxied version?

Reply:
There's also a way to define different sets of dependencies for install, so that we could e.g. have ``pip install altair_data_server[proxied]`` that would include the server proxy requirement. I don't recall offhand how that's done, but I think it would be the best approach here.

Reply:
One other thing I need to clarify here... do you know if it matters whether the kernel itself has the proxy installed, or is it that the kernel manager process needs the proxy?

Reply:
you're right. The host aka the jupyter(lab) server needs it, which in many cases is not even the same virtual environment. I'm going to remove the dependency and will adjust the README to mention it.

Reply:
Perfect.